packages/storybook/stories/CallComposite/CallCompositeDocs.tsx

```diff
@@ -271,7 +271,55 @@ export const Docs: () => JSX.Element = () => {
         gallery](./?path=/docs/ui-components-videogallery--video-gallery) component docs for more information on our
         local video tile and some of the other options we have for the local video tile when just using the components.
       </Description>
-
+      <Heading>Customizing the Gallery Layout</Heading>
```

Comment:
Nit: mention default
```suggestion
      <Heading>Customizing the default Gallery Layout</Heading>
```